<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPositioner</class>
 <widget class="QWidget" name="DialogPositioner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>agtv - Positioner</string>
  </property>
  <property name="windowIcon">
   <iconset resource="agtv.qrc">
    <normaloff>:/images/agtv_32.png</normaloff>:/images/agtv_32.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>912</width>
       <height>423</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/bg3.png);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="7" column="0" colspan="8">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Position Name</string>
        </property>
       </widget>
      </item>
      <item row="8" column="4">
       <widget class="QPushButton" name="pushButtonLoadPos">
        <property name="text">
         <string>Load Position</string>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QPushButton" name="pushButtonSavePos">
        <property name="text">
         <string>Save Position</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QComboBox" name="comboBoxPosName">
        <property name="contextMenuPolicy">
         <enum>Qt::PreventContextMenu</enum>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QPushButton" name="pushButtonDeletePos">
        <property name="text">
         <string>Delete Position</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="8">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="8">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="8">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Height</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>XOffset</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="lineEditH"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>DPI Aware</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QLineEdit" name="lineEditX"/>
      </item>
      <item row="4" column="5">
       <widget class="QLineEdit" name="lineEditW"/>
      </item>
      <item row="6" column="4">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>YOffset</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="QLineEdit" name="lineEditY"/>
      </item>
      <item row="11" column="3">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="checkBoxDpiAware">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QPushButton" name="pushButtonCurrentPos">
        <property name="text">
         <string>Current Position</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="agtv.qrc"/>
 </resources>
 <connections/>
</ui>
